src/app/auth/services/auth.service.test.ts

```diff
@@ -3,19 +3,15 @@
  */
 
 import * as authService from './auth.service';
-import { userActions } from 'app/store/slices/user';
+import { userActions } from '../../store/slices/user';
 import { UserSettings } from '@internxt/sdk/dist/shared/types/userSettings';
-import localStorageService from 'app/core/services/local-storage.service';
-import * as keysService from 'app/crypto/services/keys.service';
+import localStorageService from '../../core/services/local-storage.service';
+import * as keysService from '../../crypto/services/keys.service';
 import { AuthenticateUserParams } from './auth.service';
-import { AuthMethodTypes } from 'app/payment/types';
+import { AuthMethodTypes } from '../../payment/types';
+import { vi, describe, it, expect, beforeEach, afterEach } from 'vitest';
 
-jest.mock('../../../WebWorker');
```

Comment:
@sg-gs , I don't know how to convert this test to vitest yet. It fails the moment any call to authService appears. Seems to be something with the mock, but I'm not sure what. @xabg2  tried as well, but so far it's not working